src/components/oscal-utils/OSCALRestUtils.js

```diff
@@ -184,22 +186,51 @@ export function updateSspControlImplementationImplementedRequirementStatementByC
 ) {
   const partialRestImplementedRequirement = deepClone(implementedRequirement);
 
-  // Find our statementByComponent and update the description
-  const statementByComponent = partialRestImplementedRequirement.statements
-    .find((element) => element["statement-id"] === statementId)
-    ["by-components"].find(
-      (element) => element["component-uuid"] === componentId
-    );
+  // Find our statement
+  let statement = partialRestImplementedRequirement?.statements?.find(
```

Comment:
```suggestion
  let statement = partialRestImplementedRequirement.statements?.find(
```

The rest of this function just assumes `partialRestImplementedRequirement` itself is defined so we should start with that assumption or we should explicitly handle the case where it isn't/

Reply:
Done.